### DISCUSSION TITLE
Consider moving or allowing to override Selectable Assets Conditions into the entry types settings to allow better re-use of fields

### DISCUSSION THREAD

**Discussion by terryupton** [craftcms/cms#12395] [Ideas] [answered]
Currently you can set Selectable Assets Conditions on a field type but these are fixed for the field type.

<img width="1601" alt="Screenshot 2022-11-29 at 14 08 33" src="https://user-images.githubusercontent.com/5698596/204551650-93def32d-7b4e-4bc2-a18e-2cd2d299734b.png">

This potentially limits the reuse of the field across different entry types/sections.
it would be extremely useful this was set on the field in the entry-type field setup, or at least thew ability to over-rise the default settings like with label and instructions. 

<img width="1629" alt="Screenshot 2022-11-29 at 14 08 52" src="https://user-images.githubusercontent.com/5698596/204551666-2d4d01a8-94ad-43b5-a3cb-689ca6edc2a4.png">

A few examples I have come up against...
**Banner Images Field**
So these all use the same asset folder (Images) but on some pages the layout is designed to accomodate and must contain 3 banner images. On other sections the design allows flexibility of either 4, 5 or 6 images. It would be useful to be able to re-use the field across all of these sections. Currently I would need to create two separate fields to set these conditions independently as required. If this was done at entry-type level,  I could use the same field (name, structure, asset folder etc), but set the specific conditional requirements of the field for that entry-type/layout. 

I could see a similar requirement occurring for other fields such as related content as well.

**Comment by brandonkelly** [comment, marked as answer]
Closing this in favor of #8688.
